### ISSUE TITLE
fsvs -V does not show version number

### ISSUE THREAD

**Issue by ryandesign** [phmarek/fsvs#25]
The manpage says that `fsvs -V` will show the version number but it does not:

```
FSVS (licensed under the GPLv3), (C) by Ph. Marek; version 
```

I have built fsvs 1.2.12 from source using the tarball from https://github.com/phmarek/fsvs/archive/refs/tags/fsvs-1.2.12.tar.gz since you do not offer release downloads (#22).

I notice the build log contains this output:

```
fatal: not a git repository (or any of the parent directories): .git
/bin/sh: tags: No such file or directory
     tags
```

I notice that the Makefile uses `git` to get the version:

https://github.com/phmarek/fsvs/blob/a311665120b2e18a8b529a431ac8582892fa579c/src/Makefile.in#L14

Since I am not using a git clone that will not work.

**Comment by ryandesign** [comment]
Building with `make VERSION=1.2.12` works but I shouldn't have to do that:

```
% fsvs -V
FSVS (licensed under the GPLv3), (C) by Ph. Marek; version 1.2.12
```

**Comment by phmarek** [comment]
Should be fixed via #22.
